zubhub_frontend/zubhub/public/index.html

```diff
@@ -47,6 +47,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Zubhub</title>
+    <script src="https://kit.fontawesome.com/58aa8d2b83.js" crossorigin="anonymous"></script>
```

Comment:
Can we avoid using this and reuse parts that we already are using? Check imagePreview feature out that was done by @souad988 for the activity feature :) https://github.com/unstructuredstudio/zubhub/blob/master/zubhub_frontend/zubhub/src/assets/js/styles/views/create_activity/createActivityStyles.js#L51-L69